.drone.yml

```diff
@@ -0,0 +1,527 @@
+---
+kind: pipeline
+name: default
+type: kubernetes
+
+environment:
+  APP_NAME: ima
+  PROD_ENV: ima
+  STG_ENV: ima-stg
+  UAT_ENV: sas-ima-uat
```

Comment:
should this be ima-uat?

Reply:
Looking at [IMB-8](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-8), it is now sas-ima-uat